docs/source-1.0/spec/core/stream-traits.rst

```diff
@@ -60,6 +60,20 @@ Validation
     blob StreamingBlob
 
 
+Deserializing streaming blobs
+=============================
+
+It is often impossible for a server to know if a payload was sent by the
+client that contains zero bytes, or if no payload was explicitly sent by the
+client. As a result, required streaming members SHOULD be interpreted by
+deserializers as if they have a default value of zero bytes.
```

Comment:
What does it mean to have a default value of zero bytes? Zero bytes refers to a length of data, but an interpreted member value would have a non-zero length so does this mean interpret the value as if all (necessary) bytes for the type were set to zero? I think this needs to be re-worded to be unambiguous.